elements/pfe-primary-detail/README.md

```diff
@@ -67,10 +67,6 @@ For this component to work, there should be an equal number of `details-nav` and
 - `details-nav--header`: In case content needs to be added at the top of the nav area. Will not be matched up with `details` content.
 - `details-nav--footer`: In case contnet needs to be added at the bottom of the nav. Will not be matched up with `details` content.
 
-## Attributes
```

Comment:
@wesruv I didn't see this attribute in the JS code for the component - is it okay if I remove this from the docs?

Reply:
It's controlled with CSS, no JS necessary. Should stay in, could change it to a class if it's better suited as a class?

Reply:
Oh!  I looked and didn't see it in the CSS.  An attribute is great, my bad for missing how it was being implemented.

Reply:
Just pushed up the update reverting this change